apps/nextjs/src/pages/posts-and-comments.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import { api } from "~/utils/api";
+import Post from "../components/post";
+
+function PostsAndComments() {
+  const posts = api.post.getPosts.useQuery().data!;
+  const allComments = api.comment.getComments.useQuery().data!;
```

Comment:
generally would recommend not using `useQuery().data`, and would just use `posts.data` instead